README.md

```diff
@@ -1,14 +1,27 @@
-Todo: copy old README.md
+# Orchestrator ui library
 
-# Getting started
+This repo contains the generally reusable parts of the orchestrator ui grouped and exposed as pages, components and elements such as Icons.
+It is meant to be used together with an app that includes this library through NPM. For ease of development we have added the orchestrator example app implementation as a submodule in the folder /aps/wfo-ui.
 
--   set `AUTH_ACTIVE` env variable to false or use setup below with auth.
+To install and run the app
 
 ```
-npm
-turbo dev
+git clone git@github.com:workfloworchestrator/orchestrator-ui-library.git
+git submodule init
+git submodule update
+cp apps/wfo-ui/.env.example apps/wfo-iu/.env
+# change the values in the env file to point to your orchestrator backend
+# set auth=false or follow the directions bellow this sections
+npm i
```

Comment:
I am getting this error: 
![image](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/20791917/1923ea4a-1a3e-4b88-9b92-69ea3b491652)